source/clustering.md

```diff
@@ -291,7 +291,7 @@ have.
 ```{code-cell} ipython3
 :tags: [remove-cell]
 
-clus = data[data["cluster"] == 2][["bill_length_standardized", "flipper_length_standardized"]]
+clus = data[data["cluster"] == 0][["bill_length_standardized", "flipper_length_standardized"]]
```

Comment:
This is the reason for https://github.com/UBC-DSCI/introduction-to-datascience-python/issues/258